earth_enterprise/src/fusion/autoingest/AssetGenDaemonCpp.pl

```diff
@@ -200,7 +147,12 @@ ()
         $formalcachedinputarg
         $formalExtraUpdateArg)
 {
-    Mutable${name}AssetD asset = FindMake(ref_ $forwardtypearg,
+EOF
```

Comment:
Is there a need for these 3 lines that have been added? I'd prefer to remove them if possible.

Reply:
remnant of some debugging i had in there, removed